gobblin-iceberg/src/test/java/org/apache/gobblin/iceberg/writer/HiveMetadataWriterTest.java

```diff
@@ -350,6 +403,34 @@ private String writeRecord(File file) throws IOException {
     return path;
   }
 
+  /**
+   * Test class for exposing methods for exposing internal HiveMetaWriter functions without making them public.
```

Comment:
Typo

"Test class ~~for exposing methods~~ for exposing internal HiveMetaWriter functions"